lib/executable-config/inject/src/Obelisk/ExecutableConfig/Inject.hs

```diff
@@ -22,7 +22,7 @@ inject key = do
 -- whose value is the provided configuration
 injectPure :: DomBuilder t m => Text -> Text -> m ()
 injectPure key value =
-  let attrs = ("type" =: "text/plain" <> "id" =: ("config-" <> key) <> "data-executable-config-inject" =: "1")
+  let attrs = ("type" =: "text/plain" <> "id" =: ("config-" <> key) <> "class" =: "obelisk-executable-config-inject")
```

Comment:
I'm not 100% on this change -- on the one hand it does make the code simpler, otoh, shouldn't we leave the class namespace for the real stuff on the page to use?  Or at least we should document that we reserve classes starting with 'obelisk-' ...

Reply:
I mean, by that same logic we would need to reserve the `config-*` IDs for the client code as well.

Reply:
`data-*` attrs were added largely so that you wouldn't feel the need to overload `class` for everything. Why not just use them for their intended purpose?

Reply:
@3noch Hmm, I wasn't aware of those before. A cursory search brought this up: https://intu.io/blog/dont-use-data-attributes-to-find-html-elements-with-js/

Reply:
That author's arguments both for and against using data attributes as locators seem pretty weak.

Reply:
I agree that this argument is pretty arbitrary. The point is we can avoid stealing class namespace from designers and make it quite clear to the DOM that we are doing something highly custom with these nodes.

Reply:
Is there an API for looking up elements by these attributes? I couldn't find any.

Reply:
`document.querySelectorAll('[data-foo]');`

Reply:
To be clear, I don't care that much...just more FYI type interaction.

Reply:
Nah that's cool, I'll use that (and for the keys as well).